### PR TITLE
Version 5 GitHub Actions Support

### LINKED ISSUE #175: Restore Theme Variants
Cleanup the fallout of v4.X and migrating back to the stable theme variants

- [x] Remove configuration code. (keep update notifications and stuff)

- [x] Migrate gradle tasks to build theme variants.

- [x] Update documentation/changelog and images.

### PR DESCRIPTION
**Pull request by Unthrottled**
## Motivation

Fixes #175 

## Changes
- Added settings spotlight border color.
    - Color of the border around highlighted items when searching in the Settings dialog
- Non-Functional changes:
    - Cleaned up documentation and restored old images
    - Updated build process to create themes when the gradle task to run the IDE or build the plugin gets run.
    - Removed log noise, by getting Sentry config at error reporting time. I added configurations to my infrastructure. 

## Screenshots

Spotlight Border

## Extra Notes

I'd like to release this coming weekend, I don't like publishing a lot of updates frequently that don't add many functional user changes.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -26,6 +27,11 @@ configurations {
 intellij {
   version 'LATEST-EAP-SNAPSHOT'
   alternativeIdePath  project.hasProperty("idePath") ? project.findProperty("idePath") : ""
+
+// Convenience tool for development to include other plugins
+//  setPlugins(
+//    'indent-rainbow.indent-rainbow:1.5.1'
+//  )
```
Comment (mskelton) [inline build.gradle]: May want to throw this in a conditional block to safeguard against uncommenting it and forgetting to re-comment before releasing.

File: buildSrc/src/main/kotlin/themes/GroupStyling.kt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.markskelton.settings
+package themes
```
Comment (mskelton) [inline buildSrc/src/main/kotlin/themes/GroupStyling.kt:1]: Oh the sadness 😆

File: buildSrc/templates/oneDark.template.theme.json [hunk 3]
```diff
@@ -1,9 +1,11 @@
-
 {
   "name": "One Dark",
   "dark": true,
   "author": "Mark Skelton",
   "editorScheme": "/themes/one_dark.xml",
+  "colors": {
+    "accentColor": "#568AF2"
+  },
```
Comment (mskelton) [inline buildSrc/templates/oneDark.template.theme.json:8]: This is pretty neat

File: src/main/kotlin/com/markskelton/OneDarkTheme.kt [hunk 4]
```diff
@@ -3,18 +3,10 @@ package com.markskelton
...
 class OneDarkTheme : StartupActivity, DumbAware {
 
-  init {
-    Sentry.init("https://cb598170e51a44adbf0079abe2d79624@o403546.ingest.sentry.io/5267019?maxmessagelength=50000")
-  }
-
-
   override fun runActivity(project: Project) {
-    LegacyMigration.migrateIfNecessary()
     OneDarkThemeManager.registerStartup(project)
```
Comment (mskelton) [inline src/main/kotlin/com/markskelton/OneDarkTheme.kt:10]: Is this needed anymore?

File: buildSrc/src/main/kotlin/themes/GroupStyling.kt (hunk 2 above)
Comment (Unthrottled) [reply buildSrc/src/main/kotlin/themes/GroupStyling.kt:1]: I'll be back

File: buildSrc/templates/oneDark.template.theme.json (hunk 3 above)
Comment (Unthrottled) [reply buildSrc/templates/oneDark.template.theme.json:8]: Yeah! I started to do it with more colors but there was a lot. So I just stuck with this for the time being.

File: build.gradle (hunk 1 above)
Comment (Unthrottled) [reply build.gradle]: Good Idea!

File: src/main/kotlin/com/markskelton/OneDarkTheme.kt (hunk 4 above)
Comment (Unthrottled) [reply src/main/kotlin/com/markskelton/OneDarkTheme.kt:10]: I thought I would keep it around to show the user update notifications :shrug:

Comment (mskelton) [reply src/main/kotlin/com/markskelton/OneDarkTheme.kt:10]: Sound good.

**Review by mskelton — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (Unthrottled) [reply build.gradle]: Done!

**Comment by Unthrottled** [comment]
I kind of want to wait until there is something of substance to give to the users. I feel kind of bad releasing non-user facing changes.

**Comment by Unthrottled** [comment]
- [ ] Update all collaborators

**Comment by mskelton** [comment]
You could merge this in and just hold off on the release until later.

**Comment by Unthrottled** [comment]
Sounds like a plan!

**Pull request merged**